css/css-ui/tools/appearance-build-webkit-reftests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import os
+import re
+
+re_testname = re.compile(r"^appearance-.+\d\d\d\.html$")
+re_link_match = re.compile(r'<link rel="(mis)?match"')
+
+files = {}
+
+parentdir = os.path.abspath(os.pardir)
+assert parentdir.endswith('/css-ui')
+
+for filename in os.listdir(parentdir):
+    path = parentdir + '/' + filename
+    if os.path.isfile(path) and re_testname.search(filename) and not filename in files:
+        with open(path, "r") as file:
+            files[filename] = file.read()
+
+warning = """<!-- DO NOT EDIT THIS FILE.
+Edit the appearance-* file instead and then run:
+    cd tools
+    python3 appearance-build-webkit-reftests.py
```

Comment:
Why not `./appearance-build-webkit-reftests.py`? The script does not need the `#!` otherwise.

Reply:
Fixed